### PR TITLE
Download artifacts script to use as binary.

### LINKED ISSUE #132: Create download_artifacts.py
Similar to `run_crawler.py` this can help to use `latest_cov_build.py`.

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #132

### CODE REVIEW

File: download_artifacts.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+from coverage_crawler import latest_cov_build
+
+latest_cov_build.download_artifacts()
```
Comment (marco-c) [inline download_artifacts.py]: We should move the argparse interface from latest_cov_build.py to here too.

Comment (rhcu) [reply download_artifacts.py]: Ok, I'll update this and the other PR on the weekends. :)

**Comment by rhcu** [comment]
Will it be possible to import it then? It is not located in `coverage_crawler` folder.

**Comment by rhcu** [comment]
Oh, yep, sorry. I'll remove it now. :)

**Pull request merged**